### PR TITLE
Use the correct url for the npmjs public registry

### PR DESCRIPTION
**Pull request by token-cjg**
## Description

Use the correct url for the npmjs public registry.

## Checklist

- [ ] :guardsman: includes new unit and functional tests

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -11,6 +11,6 @@ yarn install
 # ./scripts/generate_dev_docs.sh
 
 # Remove beta once we are out of it
-npx lerna publish --conventional-commits --yes --preid 'beta' --@zendesk:registry=https://npmjs.com
+npx lerna publish --conventional-commits --yes --preid 'beta' --@zendesk:registry=https://registry.npmjs.com
```
Comment (Frederick888) [inline scripts/release.sh:14]: Does this actually work? I thought it should be `--registry=https://registry.npmjs.org`

Comment (token-cjg) [reply scripts/release.sh]: According to https://stackoverflow.com/a/67691043 one should be able to do this.

Comment (Frederick888) [reply scripts/release.sh]: I've seen things that work for npm/yarn but not lerna. Please test it out. Thanks.

Comment (token-cjg) [reply scripts/release.sh]: Yes, you are correct. This doesn't work.

**Pull request closed without merging**